### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by jbunce**
Add Dockerfile to this branch

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+RUN groupadd --gid "${GID}" "${USER}" && \
+    useradd \
+      --uid ${UID} \
+      --gid ${GID} \
+      --create-home \
+      --shell /bin/bash \
+      ${USER}
+
+ARG BUILD_PACKAGES="build-essential autoconf libtool libgmp3-dev libssl-dev wget gettext"
```
Comment (paulperegud) [inline Dockerfile]: libgmp3-dev -> libgmp-dev. We need to update the readme.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+USER plasma
+
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8 
+
+WORKDIR /home/plasma/elixir-omg/
+
+RUN sudo -H pip3 install --upgrade pip \
+  && sudo -H -n ln -s /usr/bin/python3 python \
```
Comment (paulperegud) [inline Dockerfile]: This one is a bit strange. Maybe update-alternatives instead?

File: Dockerfile (hunk 1 above)
Comment (jbunce) [reply Dockerfile]: ack

File: Dockerfile (hunk 2 above)
Comment (jbunce) [reply Dockerfile]: I'll look into that

**Pull request merged**